docs/upgrades/locks/ez_40.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: ez_40
+---
+
+## Description
+
+Halperyon Systems' EZ 40. Wait, I can't remember if 1 is prime or not.
+
+### Stats
+
+This upgrade has no additional stats and only spawns at 'kiddie' rarity.
+
+## Behavior
+
+When encountered, the ez_40 will ask for a string on its first parameter, and a digit (number) on its second parameter.
+
+<details>
+<summary>Spoilers for ez_40's parameters</summary>
```

Comment:
```suggestion
```

parameter info is not in the spoiler policy -- should not be in a spoiler block.